src/modules/tests/tests.actions.ts

```diff
@@ -102,6 +104,11 @@ export class SendCurrentTestFailure implements Action {
   constructor(public isDuplicateUpload: boolean) { }
 }
 
+export class AddCompletedTests implements Action {
+  readonly type = ADD_COMPLETED_TESTS;
+  constructor(public references: number[]) {}
```

Comment:
are these "references" the application references?

Reply:
`AddCompletedTests` is no longer present